app/src/main/java/ch/sdp/vibester/database/UsersRepo.kt

```diff
@@ -19,16 +19,30 @@ class UsersRepo @Inject constructor() {
     /**
      * This function updates a specific field of a user in the database
      * @param userID the id of the user which is being updated
-     * @param newVal the new value of the field that is being updated
+     * @param newVal (String) the new value of the field that is being updated
      * @param fieldName the field name of the field that is being updated
      */
-    fun updateField(userID: String, newVal: String, fieldName: String) {
+    fun updateFieldString(userID: String, newVal: String, fieldName: String) {
         dbRef.child(userID) //For now ID is hardcoded, will generate it creating new users next week "testUser"
             .child(fieldName)
             .setValue(newVal)
     }
 
     /**
+     * The users class which handled all the interactions with the database that are linked to users
+     * @param userID the id of the user which is being updated
+     * @param newVal (Int) the new value of the field that is being updated
+     * @param fieldName the field name of the field that is being updated
+     */
+    fun updateFieldInt(userID: String, newVal: Int, fieldName: String) {
```

Comment:
Maybe we can pass newVal as a generic type and have one function? What do you think?

Reply:
Like in this example
![image](https://user-images.githubusercontent.com/40314248/162323646-6ac2661c-7cd5-4842-b64a-91af0a3fb8a5.png)

Reply:
I think that might be a good idea for a future refactor. Like this is not important yet would be done to improve maintainability.